src/main/java/com/kobylynskyi/graphql/codegen/model/GeneratedInformation.java

```diff
@@ -0,0 +1,46 @@
+package com.kobylynskyi.graphql.codegen.model;
+
+import java.time.format.DateTimeFormatter;
+
+public class GeneratedInformation {
+
+    public static final DateTimeFormatter DATE_TIME_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ssZ");
+
+    private DateTimeGenerator dateTimeGenerator;
+    private final String generatedType;
+
+    public GeneratedInformation() {
+        this(new DefaultDateTimeGenerator());
```

Comment:
```suggestion
        this(ZonedDateTime::now);
```

If this is the only use of `DefaultDateTimeGenerator`, then maybe remove it altogeher.